config/pcl.yml

```diff
@@ -4,21 +4,41 @@ idspace: PCL
 base_url: /obo/pcl
 
 products:
-- pcl.owl: https://raw.githubusercontent.com/obophenotype/provisional_cell_ontology/master/pcl-base.owl
-- pcl.obo: https://raw.githubusercontent.com/obophenotype/provisional_cell_ontology/master/pcl-base.obo
+- pcl.owl: https://github.com/obophenotype/provisional_cell_ontology/releases/latest/download/pcl.owl
+- pcl.obo: https://github.com/obophenotype/provisional_cell_ontology/releases/latest/download/pcl.obo
+- pcl-base.owl: https://github.com/obophenotype/provisional_cell_ontology/releases/latest/download/pcl-base.owl
```

Comment:
These `pcl-base.*` files should not be under `products`, only `pcl.*` files with your IDSPACE.

Reply:
fixed, thank you